src/lib/y2partitioner/widgets/format_and_mount.rb

```diff
@@ -667,8 +672,53 @@ def items
       end
     end
 
+    # Partition identifier selector or empty widget if changing the partition id
+    # is not possible
+    class PartitionId < CWM::CustomWidget
+      # @param controller [Sequences::FilesystemController]
+      def initialize(controller)
+        @controller = controller
+        @selector = PartitionIdComboBox.new(controller) if controller.partition_id_supported?
+      end
+
+      # @macro seeAbstractWidget
+      def contents
+        if @selector
+          HBox(HSpacing(4), Left(@selector))
+        else
+          Empty()
+        end
+      end
+
+      # Synchronize the widget with the information from the controller
+      def refresh
+        @selector.refresh if @selector
+      end
+
+      # @macro seeAbstractWidget
+      def enable
+        @selector.enable if @selector
+      end
+
+      # @macro seeAbstractWidget
+      def disable
+        @selector.enable if @selector
```

Comment:
Should it be `@selector.disable` ?

Reply:
Indeed. Checked and I introduced a small bug. Fixing